### PR TITLE
Update E2E Describe() Blocks

### PR DESCRIPTION
**Pull request by samuelfreiberg**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [X] win32 (Office)
- [X] windows
- [ ] android

### Description of changes

Describe() blocks should not have an async function as the 2nd parameter according to Jasmine docs. I'm normalizing the pattern, changing `async () =>` and `function ()` to `() => {`

Jasmine Describe() docs: https://jasmine.github.io/tutorials/your_first_suite
Similar bug: https://github.com/jasmine/jasmine/issues/1487

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/E2E/wdio.conf.windows.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ exports.config = {
    * @param {Array.<Object>} capabilities list of capabilities details
    * @param {Array.<String>} specs List of spec file paths that are to be run
    */
-  before: async function () {
+  before: async () => {
```
Comment (ksiler) [inline apps/E2E/wdio.conf.windows.js:125]: Should this (as well as the other cases/comments in the other wdio.conf files) be changed too?

**Comment by ksiler** [comment]
Could you provide a link to the Jasmine docs where this pattern is described in the PR description?

**Comment by samuelfreiberg** [comment]
> Could you provide a link to the Jasmine docs where this pattern is described in the PR description?

Added

File: apps/E2E/wdio.conf.windows.js (hunk 1 above)
Comment (samuelfreiberg) [reply apps/E2E/wdio.conf.windows.js:125]: Updated all WDIO files with this pattern. I don't think the commented ones matter atm

**Review by ksiler — APPROVED** [review]

**Pull request merged**